### PR TITLE
Added translation.

### PR DESCRIPTION
**Pull request by nosferatu500**
Added translated copies of files on Russian language.

@tomasherceg

### CODE REVIEW

**Comment by tomasherceg** [comment]
Thak you very much. We'll have to add the support for localization because currently there is no way to switch the language. 

Also, there are error messages in the resx files with text that appear when the code provided by the user is not correct. Could you please translate them as well?

And finally, the home page is not localizable yet - we'll add a RESX file for static texts on the web site asap to allow to localize the whole app.

Many thanks.

**Comment by nosferatu500** [comment]
@tomasherceg Well, I will translate error messages. I didn't begin to translate the home page as there all and is so clear :), but if you want to realize a possibility of full localization, then I only for). So I will translate also her too.

If suddenly I have passed something somewhere else - report, I will translate.

**Comment by nosferatu500** [comment]
Add the translation for the main page, and error messages

File: DotvvmAcademy/DotvvmAcademy/Lessons/Rus/Lesson4.xml [hunk 1]
```diff
@@ -0,0 +1,626 @@
...
+		<!--9-->
+		<Step Type="Info">
+			<!--Required element-->
+			<Title>Partial Validation</Title>
+			<!--Required element-->
+			<Description>
+				<![CDATA[
+По умолчанию все кнопки запускают проверку на всей ViewModel, и если есть какая-либо ошибка, то команда в ViewModel не выполняется вообще.
+
+Тем не менее, ViewModel часто содержит некоторые дочерние объекты и вы не хотели бы, проверять их все.
```
Comment (MDuzij) [inline DotvvmAcademy/DotvvmAcademy/Lessons/Rus/Lesson4.xml]: ...дочерние объекты и вы не хотели бы проверять их все.

File: DotvvmAcademy/DotvvmAcademy/Lessons/Rus/Lesson4.xml [hunk 2]
```diff
@@ -0,0 +1,626 @@
...
+		<!--11-->
+		<Step Type="CsharpCode">
+			<!--Required element-->
+			<Title>Add ViewModel Properties</Title>
+			<!--Required element-->
+			<ValidationKey>Lesson4Step11Validator</ValidationKey>
+			<!--Required element-->
+			<Description>
+				<![CDATA[
+Объявите `SubscriptionFrom` и `SubscriptionTo` типа `DateTime` в viewmodel.
```
Comment (MDuzij) [inline DotvvmAcademy/DotvvmAcademy/Lessons/Rus/Lesson4.xml]: Объявите свойства `SubscriptionFrom` и `SubscriptionTo` типа `DateTime` в viewmodel.

**Pull request merged**